### PR TITLE
[nnstreamer/apptest] update env variables

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Before adding a script to build nnstreamer plugins, set env variables for nnstreamer.

TODO :
We have to build (or download) nnstreamer plugins later.

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 1]
```diff
@@ -104,7 +112,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     fi
 
     # Build a source code
-    meson --buildtype=plain --werror --prefix=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
+    meson --prefix=${NNST_ROOT} --sysconfdir=${NNST_ROOT} --libdir=lib --bindir=bin --includedir=include build
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: :+1:

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 2]
```diff
@@ -91,6 +91,14 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     echo -e "[DEBUG] LD_LIBRARY_PATH is '$LD_LIBRARY_PATH'"
     echo -e "[DEBUG] GST_PLUGIN_PATH is '$GST_PLUGIN_PATH'"
 
+    # nnstreamer env variables and include paths
+    export NNSTREAMER_CONF=$NNST_ROOT
+    export NNSTREAMER_FILTERS=$NNST_ROOT/lib/nnstreamer/filters
+    export NNSTREAMER_DECODERS=$NNST_ROOT/lib/nnstreamer/decoders
+    export NNSTREAMER_CUSTOMFILTERS=$NNST_ROOT/lib/nnstreamer/customfilters
+    export C_INCLUDE_PATH=$C_INCLUDE_PATH:$NNST_ROOT/include
+    export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$NNST_ROOT/include
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: How about we use the below naming style. 
**s/CPLUS/CPP/g**

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 3]
```diff
@@ -259,7 +267,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     echo -e "[DEBUG] -------------------- xauth: end   ----------------------"
 
 
-    # App (Consumer1): Test /dev/video0 status with gst-lanch-1.0 command 
+    # App (Consumer1): Test /dev/video0 status with gst-lanch-1.0 command
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Nice catch.

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh (hunk 2 above)
Comment (jaeyun-jung) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: it is predefined name to search inc files (https://gcc.gnu.org/onlinedocs/cpp/Environment-Variables.html)

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Got it. Thanks.

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: https://github.com/nnsuite/nnstreamer-example/issues/14

**Pull request merged**